<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeAvatarForm</class>
 <widget class="QWidget" name="ChangeAvatarForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="backgroundWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>420</width>
     <height>670</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#backgroundWidget
{
	background-color:#474a59;
	border-radius:15px;
}</string>
   </property>
   <widget class="QPushButton" name="closeBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#closeBtn{

            border-radius: 5px;
}
#closeBtn:hover
{
            background-color: rgba(255,255,255,.3);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Asserts/icons/close.svg</normaloff>:/Asserts/icons/close.svg</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="friendName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#c9c9c9;</string>
    </property>
    <property name="text">
     <string>更换头像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="underLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>370</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#underLine
{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #e38283 ,stop:0.33 #c19f9e, stop:0.67 #91b8bb,stop:1 #1bced8);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>69</y>
      <width>420</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_upload">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_upload
{
	background-color:transparent;
	color:#fff;
	border:2px solid #fff;
	border-radius:5px;
}
#pushButton_upload:hover
{
	background-color:#fff;
	color:#000;
}</string>
     </property>
     <property name="text">
      <string>上传头像</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>361</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="styleSheet">
         <string notr="true">#horizontalSlider
{
	height:50px;
}
#horizontalSlider::groove
{
	background-color:rgba(255, 255, 255, 128);
	height:3px;
}
#horizontalSlider::handle
{
	background-color:#c9c9c9;
	width: 10px;
	height: 10px;
	margin: -6px 5px -6px 5px;
	border-radius:5px; 
	border: 3px solid #ffffff;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clock">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/Asserts/icons/rotate_clock_normal.png</normaloff>:/Asserts/icons/rotate_clock_normal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_anti">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/Asserts/icons/rotate_anti_normal.png</normaloff>:/Asserts/icons/rotate_anti_normal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>540</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_cancel
{
	background-color:transparent;
	color:#fff;
	border:2px solid #fff;
	border-radius:5px;
}
#pushButton_cancel:hover
{
	background-color:#fff;
	color:#000;
}</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>540</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_ok
{
	background-color:transparent;
	color:#fff;
	border:2px solid #fff;
	border-radius:5px;
}
#pushButton_ok:hover
{
	background-color:rgba(32, 202, 100,0.5);
	color:#fff;
}</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="underLine_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>370</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#underLine_2
{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #e38283 ,stop:0.33 #c19f9e, stop:0.67 #91b8bb,stop:1 #1bced8);

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DragScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>360</width>
       <height>360</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>358</width>
        <height>358</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_avatar">
         <property name="styleSheet">
          <string notr="true">border-radius:5px;
border:2px solid #fff;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>pushButton_upload</zorder>
    <zorder>pushButton_upload</zorder>
    <zorder>horizontalSlider</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>pushButton_cancel</zorder>
    <zorder>pushButton_ok</zorder>
    <zorder>underLine_2</zorder>
    <zorder>scrollArea</zorder>
   </widget>
   <zorder>closeBtn</zorder>
   <zorder>friendName</zorder>
   <zorder>underLine</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>mainWidget</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">dragscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
